portfolio/src/main/java/com/google/sps/servlets/Email.java

```diff
@@ -0,0 +1,18 @@
+package main.java.com.google.sps.servlets;
+
+/**
+ * Defines a class to describe an email and it's components
+ */
+
+ public class Email{
+
+    public long id;
+    public String title;
+    public long timestamp;
```

Comment:
Can you add comments on what each of these fields represent?

Reply:
Yes I added comments